standalone-metastore/metastore-server/src/test/java/org/apache/hadoop/hive/metastore/properties/HMSJsonClientSslTest.java

```diff
@@ -92,30 +89,34 @@ protected void stopServer(int port) throws Exception {
     }
   }
 
+  protected static String getScheme(Configuration conf) {
+    return MetastoreConf.getBoolVar(conf, MetastoreConf.ConfVars.USE_SSL) ? HTTPS : HTTP;
+  }
 
   @Override
-  protected PropertyClient createClient(Configuration conf, int sport) throws Exception {
-    String path = MetastoreConf.getVar(conf, MetastoreConf.ConfVars.PROPERTIES_SERVLET_PATH);
-    URL url = new URL("http://hive@localhost:" + sport + "/" + path + "/" + NS);
+  protected PropertyClient createClient(Configuration conf, int port) throws Exception {
+    String servletPath = MetastoreConf.getVar(conf, MetastoreConf.ConfVars.PROPERTIES_SERVLET_PATH);
```

Comment:
The local variable declaration could be replaced with a path field in several places, including in inherited test classes such as HMSHttpClientSslTest